### DISCUSSION TITLE
User who has a new PC and can't connect to server

### DISCUSSION THREAD

**Discussion by jolt06** [nextgenhealthcare/connect#5567] [Q&A]
I have a user who has a new PC and i installed MIrth connect.  the issue is that the user can't log in on the new PC. but when they use the same credentials on the old pc they are able to log in. i have even tested the credentials and we are still getting the same issue. 
![log in issue](https://user-images.githubusercontent.com/55763689/205978712-855fea72-ade4-44ed-8b7e-69e8acc3e816.jpg)
this is a old version on the new pc as a test.

**Comment by pacmano1** [comment]
Moving this to discussion until a bug or issue is confirrmed.

You need to provide far more details on your configuration.  

Operating System on both PCs.
Are you using the Admin Launcher?  If yes are you overriding the bundled java version?
If you are not using the Admin Launcher what verstion of java?

**Reply by jolt06** [reply]
both PCs are using window 10 ent 64bit and we are using the admin launcher. how would we override the java bundle?

**Reply by pacmano1** [reply]
![image](https://user-images.githubusercontent.com/44065187/205984909-a6b99a52-8692-4d69-b73d-2f9165077ace.png)

**Comment by jolt06** [comment]
![image](https://user-images.githubusercontent.com/55763689/205985990-c3a38f1a-c077-438d-833c-a9d3ef2137ac.png)
can i access the admin launcher from here?

**Reply by pacmano1** [reply]
Lost me.  You said you were using the admin launcher.  If you have not seen the application in my screenshot before, then you are not using the admin launcher.

Download it at:

https://www.nextgen.com/products-and-services/mirth-connect-downloads


I don't run mirth on windows, so I can't remember if you can download the installer from that dialog box you just posted.

I don't know for a fact that the admin launcher will fix your problem, but it is a good place to start.

**Reply by jolt06** [reply]
im still getting a login error. made a few changes on the admin launcher but no luck

**Reply by ab-20-m** [reply]
If you installed a completely new instance of Mirth Connect, out of the box that is configured to use a Derby instance. The two laptops sound like two different Mirth instances so everything is WAD.

**Reply by jolt06** [reply]
i have never used this program before and the user has used it the most but it was setup before our times at the company. so im still figuring things out on this setup.

**Reply by pacmano1** [reply]
Good point @ab-20-m , I sort of assumed there was a single instance of the mirth server process and the OP was attempting to conenct to that from some new workstation.

@jolt06 , you gotta read the manaul, not giving you a hard time.

**Reply by pacmano1** [reply]
if it was a clean install of mirth server on the new PC the username and password are "admin"

**Reply by jolt06** [reply]
ok i was able to get in with ad min. now we just recreate the workspace similar to the old pc?

**Reply by pacmano1** [reply]
You can do a full config export and re-import it to the new machine.

Again - read the user manual.

But is this a dev machine?  prod?   I would normally expect a shared instance of mirth server with your workstations connecting to that shared instance.

**Reply by jolt06** [reply]
thank you all!!!
